src/plugins/unified_search/public/filter_bar/filter_item/filter_items.tsx

```diff
@@ -29,8 +30,8 @@ export interface Props {
 const FilterItemsUI = React.memo(function FilterItemsUI(props: Props) {
   const groupRef = useRef<HTMLDivElement>(null);
   const kibana = useKibana<IDataPluginServices>();
-  const { appName, usageCollection, uiSettings } = kibana.services;
-  if (!uiSettings) return null;
+  const { appName, usageCollection } = kibana.services;
```

Comment:
It doesn't seem like `uiSettings` is used anywhere in `filter_items` or any of its children... So getting rid of it for now to clean things up. May have missed something though - feel free to let me know!